Horan_Colby/Assignments/project_tdd/project_spec.rb

```diff
@@ -0,0 +1,39 @@
+require_relative 'project'
+
+RSpec.describe Project do
+  @name = "Project Name"
+  @description = "I am a project"
+	it 'has a getter and setter for name attribute' do
+		project = Project.new("#{@name}", "#{@description}")
+		project.name = "Changed Name"
+		expect(project.name).to eq("Changed Name")
+	end
+
+  it 'has a getter and setter for description attribute' do
+    project = Project.new("#{@name}", "#{@description}")
+    project.name = "Changed Name"
+    expect(project.name).to eq("Changed Name")
+  end
+
+  it 'has to have an intializer with variables for name and description' do
+    project = Project.new("#{@name}", "#{@description}")
+    expect(project.name).to eq("#{@name}")
+    expect(project.description).to eq("#{@description}")
+  end
+
+  it 'has a method elevator_pitch to display name and description' do
+    one = Project.new("#{@name}, #{@description}")
+    two = Project.new("#{@name}, #{@description}")
+    expect(one.elevator_pitch).to eq("#{@elevator_pitch}")
+    expect(two.elevator_pitch).to eq("#{@elevator_pitch}")
+  end
+
+  ## I literally have no idea with ^^^ failed. 
```

Comment:
elevator_pitch is a method and not a variable that you can interpolate.  With .to eq you're testing against what is returned.  The elevator_pitch method above is returning self, so what the test is getting back is a Project instance.  A couple options is to just remove the self and instead of using puts, use print (this is because puts returns nil).  You're test should then look like 
expect(one.elevator_pitch).to eq("Project Name, I am a project")

The other option if you want to return self is to test for the output which would look like this:

expect{one.elevator_pitch}.to output("Project Name, I am a project").to_stdout